### DISCUSSION TITLE
Galaxy Brain help

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [Schweinepriester/github-profile-achievements#664] [Q&A] [answered]
3

**Comment by ArielMAJ** [comment, marked as answer]
I can help you.

**Reply by AnzhiZhang** [reply]
Thanks, could you create some discussions so I can answer them?

**Reply by ArielMAJ** [reply]
Sure, doing it.

**Reply by AnzhiZhang** [reply]
Thanks, I also created some questions.

**Reply by ArielMAJ** [reply]
Np! Ty! Could you also leave a start on [this repo](https://github.com/ArielMAJ/test.me)? I left you some stars as well. Ty again!

**Reply by AnzhiZhang** [reply]
I am sorry I cant help you on repo stars, I do only star repos that I really use them and they are really helpful.
Sorry for that, please feel free to remove your stars on my repo if you like.
